src/app/models/Site.ts

```diff
@@ -107,4 +120,34 @@ export class Site extends AbstractModel implements ISite {
       siteId: this.id,
     });
   }
+
+  /**
+   * Get site latitude
+   */
+  public getLatitude(): number {
+    return this.latitude ?? this.customLatitude;
+  }
+
+  /**
+   * Get site longitude
+   */
+  public getLongitude(): number {
+    return this.longitude ?? this.customLongitude;
+  }
+
```

Comment:
seems like duplicated logic with the constructor?

Reply:
Yes and no, the latitude and longitude for a model are only set if the user can actually read the un-obfuscated coordinates. For all other users, they will still use the custom values.

Reply:
The reason the constructor sets the latitude and longitude to be the same as the custom values is because the API will read the lat/lng values when changing the coordinates of the site. Setting these values for sites where the user does not actually know the correct lat/lng removes the semantic difference provided by the API.